.gitignore

```diff
@@ -1,3 +1,4 @@
+*notebooks/*
```

Comment:
What is this directory? Is it missing from git or autogenerated?

Reply:
Autogenerated.  Removed.